### ISSUE TITLE
Update "Is using aria attributes failing WCAG?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#59]
The short answer can lose the word "custom" from "custom HTML widgets."

More information is currently ...
However, it is strongly advised to avoid using aria as much as possible, and to use the default HTML tags and widgets to ensure a maximum of accessibility compatibility.

More information could be something like this ...

- Where ARIA attributes duplicate native HTML elements and attributes, use the native HTML whenever possible. Screen reader support for native HTML elements is strong.
- Where ARIA attributes introduce new features to HTML, ensure you understand how to implement them. Incorrect ARIA can break the screen reader user experience. 

[end]

Ideally we'd have a link to the course's "Everyday use ARIA" section, but that module's still in the planning stages.  The course does have a page on [ARIA live regions](https://bati-itao.github.io/learning/esdc-self-paced-web-accessibility-course/module11/aria-live.html) that could serve as an example of the second point, for the time being.
